### PR TITLE
feat(glean): Add Glean events for BentoMenu

### PR DESCRIPTION
**Pull request by vpomerleau**
## Because

- We want to track views of the bento menu and clicks on suggested product links

## This pull request

- Add new Glean events and implement in fxa-settings, add unit tests

## Issue that this pull request solves

Closes: #FXA-10219

## Checklist

_Put an `x` in the boxes that apply_

- [x] My commit is GPG signed.
- [x] If applicable, I have modified or added tests which pass locally.
- [ ] I have added necessary documentation (if appropriate).
- [ ] I have verified that my changes render correctly in RTL (if appropriate).

## Screenshots (Optional)

Please attach the screenshots of the changes made in case of change in user interface.

## Other information (Optional)

Any other information that is important to this pull request.

### CODE REVIEW

File: packages/fxa-settings/src/components/Settings/BentoMenu/index.tsx [hunk 1]
```diff
@@ -33,6 +35,13 @@ export const BentoMenu = () => {
   const { env } = useConfig();
   const ftlMsgResolver = useFtlMsgResolver();
 
+  useEffect(() => {
+    if (isRevealed && !hasLoggedViewEvent) {
+      GleanMetrics.accountPref.bentoView();
+      setHasLoggedViewEvent(true);
```
Comment (LZoog) [inline packages/fxa-settings/src/components/Settings/BentoMenu/index.tsx:41]: Is the intent that we only want to ever log 1 of these view events even if the user clicks on it, closes, and re-opens?

File: packages/fxa-shared/metrics/glean/fxa-ui-metrics.yaml [hunk 2]
```diff
@@ -2131,6 +2131,125 @@ account_pref:
     expires: never
     data_sensitivity:
       - interaction
+  bento_view:
+    type: event
+    description: |
+      User clicks on the bento menu on the top right of the account settings page
```
Comment (LZoog) [inline packages/fxa-shared/metrics/glean/fxa-ui-metrics.yaml:2137]: Not sure we want "top right" here, on RTL it's flipped.

**Review by LZoog — APPROVED** [review]
LGTM, just one question and one comment about the yaml!

File: packages/fxa-shared/metrics/glean/fxa-ui-metrics.yaml (hunk 2 above)
Comment (vpomerleau) [reply packages/fxa-shared/metrics/glean/fxa-ui-metrics.yaml:2137]: Good catch, I'd copied directly from description in ticket. I'll update!

File: packages/fxa-settings/src/components/Settings/BentoMenu/index.tsx (hunk 1 above)
Comment (vpomerleau) [reply packages/fxa-settings/src/components/Settings/BentoMenu/index.tsx:41]: That was my intent, but it might not be necessary.

**Pull request merged**